app/app/deprovision/route.js

```diff
@@ -0,0 +1,29 @@
+import Ember from "ember";
+
+export default Ember.Route.extend({
+
+  actions: {
+    deprovision: function(){
+      var app = this.currentModel;
+      var route = this;
+      var controller = this.controller;
+      var store = this.store;
+      controller.set('error', null);
+
+      app.get('stack').then(function(stack) {
+        var op = store.createRecord('operation', {
+          type: 'deprovision',
+          app: app
+        });
+        return op.save().then(function(){
+          return app.reload();
```

Comment:
this is to pick up the fact that the status of the app is now 'deprovisioning', yeah? Is that info shown in the UI anywhere?
Not strictly necessary, but the intent of the [app model STATUSES](https://github.com/aptible/diesel.aptible.com/blob/master/app/models/app.js#L4-L7) variable is to enumerate all the possible valid statuses. Might be worth adding "deprovisioning" there.